samples/logstash/es_template_accounting.json

```diff
@@ -1,77 +1,74 @@
 {
-  "version" : 1,
-  "template" : "ngx_accounting-*",
-  "settings" : {
-    "index" : {
-      "refresh_interval" : "5s"
+  "version": 1,
+  "index_patterns": ["ngx_accounting-*"],
```

Comment:
Elasticsearch 7.x throws a deprecation warning for template, telling us to use index_patterns instead.